vlasovsolver/cpu_trans_map_amr.cpp

```diff
@@ -856,15 +998,16 @@ void check_ghost_cells(const dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geometry>
          dy = mpiGrid[ids[0]]->SpatialCell::parameters[CellParams::DZ];
          break;
       case 1:
-         dx = mpiGrid[ids[0]]->SpatialCell::parameters[CellParams::DZ];
-         dy = mpiGrid[ids[0]]->SpatialCell::parameters[CellParams::DX];
+         dx = mpiGrid[ids[0]]->SpatialCell::parameters[CellParams::DX];
+         dy = mpiGrid[ids[0]]->SpatialCell::parameters[CellParams::DZ];
```

Comment:
I prefer the previous ordering. But this is not used anyway. ;)

Reply:
It's true that the dx andy values are not currently used for anything, but I want to point out that this is not a cosmetic change but actually fixes the orderings to match with what dccrg assumes.

Reply:
All right then, that reason primes over my esthetic preferences.